<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoperWindow</class>
 <widget class="QMainWindow" name="ShoperWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shoper Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>631</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="supplierManagerment">
     <attribute name="title">
      <string>Supplier Managerment</string>
     </attribute>
     <widget class="QListWidget" name="supplierList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addSupplierButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteSupplierButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="goodTable">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>441</width>
        <height>261</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Buying Price</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="deleteGoodButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>280</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addGoodButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>280</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="supplierNameInput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Supplier Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="goodNameInput">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>280</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Good Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="moreGoodButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="buyingPriceInput">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>280</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Buying Price</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="priceInput">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Price</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="goodsManagerment">
     <attribute name="title">
      <string>Goods</string>
     </attribute>
     <widget class="QTableWidget" name="goodsTable">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>10</y>
        <width>341</width>
        <height>301</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Good ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Good Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="goodsNameInput">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>123</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Good Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="goodsPriceInput">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>123</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addGoodsButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Goods</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteGoodsButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updateGoodsButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>26</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="goodsIDInput">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>123</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Good ID</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="cashierManagerment">
     <attribute name="title">
      <string>Cashier Management</string>
     </attribute>
     <widget class="QTableWidget" name="cashierTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>241</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="addCashierButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>274</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteCashierButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>274</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="cashierOutput">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>321</width>
        <height>241</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Select a cashier to show the work capacity.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cashierNameInput">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>274</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Cashier name</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="transactionRecord">
     <attribute name="title">
      <string>Transaction Record</string>
     </attribute>
     <widget class="QTextEdit" name="transactionOutput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>591</width>
        <height>271</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
